Anima/math-tools/multi_compartment_base/animaMCMConstants.h

```diff
@@ -28,4 +29,58 @@ const double DiffusionSmallDelta = 10.0e-3;
 //! Default big delta value (classical values)
 const double DiffusionBigDelta = 40.0e-3;
 
+//! Default zero lower bound (in case we want something else than zero one day)
+const double MCMZeroLowerBound = 0.0;
+
+//! Epsilon value in case we do not want to have parameters reaching their true bounds (used for now in DDI)
+const double MCMEpsilon = 1.0e-2;
+
+//! Axial diffusivity add on to lower bound (used to ensure a minimal anisotropy to the anisotropic compartments)
+const double MCMAxialDiffusivityAddonLowerBound = 5.0e-4;
+
+//! Diffusivity lower bound for estimation
+const double MCMDiffusivityLowerBound = 1.0e-5;
+
+//! Diffusivity upper bound
+const double MCMDiffusivityUpperBound = 3.0e-3;
+
+//! Radial diffusivity upper bound
+const double MCMRadialDiffusivityUpperBound = 1.0e-3;
+
+//! Free water diffusivity lower bound
+const double MCMFreeWaterDiffusivityLowerBound = 2.0e-3;
+
+//! Free water diffusivity upper bound
+const double MCMFreeWaterDiffusivityUpperBound = 4.0e-3;
+
+//! Isotropic restricted water diffusivity lower bound
```

Comment:
IRW bounds are hard to define because the underlying model assumes Gaussian isotropic diffusion which is wrong and makes the diffusivity depend on the b-value. In some cases, e.g. HCP data, that diffusivity is around 7e-5 mm2/s.

Reply:
I calculated it for HCP (with units being in seconds, Tesla and mm) and I get 7.5e-4 (I might be wrong though). Actually deltas for HCP are not that different from CUSP I believe (I still need Elise input on this). Let's say that for this compartment we want right bounds for classical one shell clinical data. IRW will be replaced by Stanisz in HCP anyway, so I think those values are ok